ex6.py

```diff
@@ -0,0 +1,7 @@
+#even or odd?
+if __name__ == "__main__": 
+    x = int(input("please enter a number: ")) #a number which given by the user
+    if x%2 == 0: #using modulo operator to classify whether x is even or odd
+        print(f'The given number is an even!')
+    else:
+        print(f'The given number is an odd!')
```

Comment:
My only problem is with the grammer: "The given number is even"
*an* is not needed.

Other than that... great work!